reference/wishlists.v3.yml

```diff
@@ -17,15 +40,11 @@ produces:
 paths:
   /wishlists:
     post:
-      description: >-
-        Creates a Wishlist and Wishlist Item. More than one item can be added in
-        the POST.
-
+      description: |-
+        Creates a wishlist and wishlist item. More than one item can be added in the POST.
 
         **Required Fields**
-
         * name
-
         * customer_id
       summary: Create a Wishlist
```

Comment:
```suggestion
      summary: Create a wishlist
```

Reply:
Changing the summary field will change the way the name of the endpoint appears on the left pane of API Reference.